packages/quantic/force-app/main/default/lwc/quanticTab/quanticTab.html

```diff
@@ -1,7 +1,15 @@
 <template>
   <template if:true={shouldDisplay}>
     <li class={tabClass} title={label} role="presentation">
-      <a class="slds-tabs_default__link" role="tab" onclick={select}>{label}</a>
+      <button
+        class="slds-tabs_button slds-button"
+        onclick={select}
+        aria-label={arialLabelText}
+      >
+        <a class="slds-tabs_default__link" role="tab"
```

Comment:
Since the parent is a button, the `tab` role is unnecessary here. I would simply remove the role for the `a` element.

Additionally, tabs seem to be meant to be part of a `tablist` and associated with a `tabpanel` ([doc](https://www.w3.org/TR/wai-aria-1.2/#tab)), which doesn't make much sense for our search pages, so I think it's simpler to use a toggle button.

Lastly, even when overwriting the `role` of an anchor element, I recall some issues being reported in JSUI about on similar components. I would replace the `a` element with a `span` or remove it.

Reply:
Also, I would avoid in general have an `a` without an `href`.

Reply:
Yeah good point I thought we'd agreed to remove the `role="tab"` but then again maybe I just lost track.

Reply:
role="tab" removed